lib/tom_queue/deferred_work_manager.rb

```diff
@@ -80,13 +80,19 @@ def schedule(response, headers, payload)
     def start
       debug "[DeferredWorkManager] Deferred process starting up"
 
+      register_signal_handlers
```

Comment:
Shall we do `Process.daemon(true)` in `deferred_work_manager_cli.rb` (might be called `daemon.rb` in the future) then? IMO It's not daemon till we have "daemonised" it